components/reddit/sources/new-hot-posts-on-a-subreddit/new-hot-posts-on-a-subreddit.js

```diff
@@ -0,0 +1,81 @@
+const common = require("../common");
+const regionData = require("./region-data");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-hot-posts-on-a-subreddit",
+  name: "New hot posts on a subreddit",
+  description:
+    "Emits an event each time a new hot post is added to the top 10 items in a subreddit.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    subreddit: {
+      propDefinition: [common.props.reddit, "subreddit"],
+    },
+    region: {
```

Comment:
Can we make this prop optional? I think the "GLOBAL" default makes sense

Reply:
Ok, made optional, GLOBAL default.